core/domain/activity_services_test.py

```diff
@@ -27,54 +27,69 @@
 from core.domain import user_services
 from core.tests import test_utils
 
+from typing import List
+from typing_extensions import Final
+
+ActivityReferenceList = List[activity_domain.ActivityReference]
```

Comment:
Do not redefine this, reuse the one from _activity_services.py_

Reply:
Done!